Misha/Files/mp3_tagger/tagger_main.py

```diff
@@ -0,0 +1,45 @@
+import eyed3
+
+
+def input_mp3_file_name(mp3_file_name):
+    file_split = str(mp3_file_name).split('.')
+    if len(file_split) == 1 or file_split[len(file_split)-1] != 'mp3':
+        raise TypeError('Invalid file type')
+    else:
+        try:
+            audiofile = eyed3.load(mp3_file_name)
+            return audiofile
+        except OSError:
+            raise OSError('file not found')
+
+
+def get_tag(audiofile):
```

Comment:
Please remove one line function

Reply:
I should have this because i havent built-in function to get tuple of ('artist', 'album', 'album artist, 'title, 'track num')